x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/api/install_prebuilt_rules_and_timelines/route.ts

```diff
@@ -103,7 +104,14 @@ export const createPrepackagedRules = async (
     await exceptionsListClient.createEndpointList();
   }
 
-  const latestPrepackagedRulesMap = await getLatestPrebuiltRules(ruleAssetsClient);
+  let latestPrepackagedRulesMap = await getLatestPrebuiltRules(ruleAssetsClient);
```

Comment:
nit: It could be a separate function like `installLatestAvailablePrebuiltRules` which will hide implementation details of fetching the latest available prebuilt rules.

Reply:
Not sure I understand what you mean. We already have the `getLatestPrebuiltRules` function that fetches the latest available rules.

Reply:
I meant the scope of this line and the next `if` block as it all serves one purpose but `if` block contains fallback functionality.